vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go

```diff
@@ -45,13 +45,14 @@ import (
 type wellKnownController string
 
 const (
+	cronJob               wellKnownController = "CronJob"
 	daemonSet             wellKnownController = "DaemonSet"
 	deployment            wellKnownController = "Deployment"
+	node                  string              = "Node"
```

Comment:
Why string and not wellKnownController here?

Reply:
Done.